backend/fms_core/models/process_sample.py

```diff
@@ -20,7 +20,7 @@
 class ProcessSample(TrackedModel):
     process = models.ForeignKey(Process, on_delete=models.PROTECT, related_name="process_sample", help_text="Process")
     source_sample = models.ForeignKey(Sample, on_delete=models.PROTECT, related_name="process_sample", help_text="Source Sample")
-    execution_date = models.DateField(default=timezone.now, help_text="Date of execution of the process.")
+    execution_date = models.DateField(help_text="Date of execution of the process.")
```

Comment:
here blank and Nil are at False by default.. by removing default=timezone.now, you now have to manually handle the entry of a default execution_date at each ProcessSample object creation. For now, I see that you added 'update_date' to the SampleUpdate Resource. However, it is still not in the template as a column, and will probably end up with a None value...